bindings/rust/aws-lc-sys-template/src/lib.rs

```diff
@@ -1,8 +1,29 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR ISC
 
-pub use bindings::*;
+#[cfg(all(not(feature = "bindgen"), target_os = "linux", target_arch = "x86"))]
+pub use linux_x86_bindings::*;
+#[cfg(all(not(feature = "bindgen"), target_os = "linux", target_arch = "x86"))]
+mod linux_x86_bindings;
```

Comment:
It's unfortunate that we have to repeat the same long `#[cfg(...)]` lines for each.  The only alternative I've for for this is the [cfg_aliases](https://crates.io/crates/cfg_aliases) crate.